### ISSUE TITLE
Please add support for infinite scrolling (both rows and columns)

### ISSUE THREAD

**Issue by rajaraodv** [tannerlinsley/react-table#186]
Please add support for infinite scrolling (both rows and columns). This feature, especially infinite columns will make this an enterprise ready table

**Comment by tannerlinsley** [comment]
TL;DR: Currently this is not in the scope of React Table, but could possibly come around in the future when the library is a bit more mature.  Closing for now.

Detail: This is probably one of the most discussed and requested features for React Table, but actually goes against much of the root cause of its existence (see [this medium article](https://medium.com/@tannerlinsley/why-i-wrote-react-table-and-the-problems-it-has-solved-for-nozzle-others-445c4e93d4a8) for an deeper discussion). Though some of the ideas discussed in that article are outdated, the opinions of scrolling are still very rooted in the library.
